picaso/fluxes.py

```diff
@@ -1348,6 +1352,9 @@ def get_reflected_1d(nlevel, wno,nwno, numg,numt, dtau, tau, w0, cosb,gcos2, fta
                                 0.75*(1+cos_theta**2.0) #rayleigh phase function
                                 )
                             )
+                elif single_phase==4:#'LAB':
```

Comment:
@colinhamill am I okay to comment this out?

Reply:
or does it add some functionality ?

Reply:
Yes, this can be commented out. This part is used for the Lab vs. DDA vs. HG study, and I forgot to remove it. It does not impact the reflected phase curve functionality.